zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/ClockValidator.java

```diff
@@ -19,8 +20,13 @@ public class ClockValidator {
   public static Optional<ProblemDetail> validateClockPinRequest(final ClockPinRequest pinRequest) {
     return validate(
         violations -> {
-          if (pinRequest.getTimestamp() == null) {
+          final Long timestamp = pinRequest.getTimestamp();
```

Comment:
🔧 To avoid if/else statement you can refactor the validation like this:

```
Optional.ofNullable(pinRequest.getTimestamp())
            .ifPresentOrElse(
                timestamp -> {
                    if (timestamp < 0) {
                        violations.add(ERROR_MESSAGE_INVALID_ATTRIBUTE_VALUE
                            .formatted("timestamp", timestamp, "not negative"));
                    }
                },
                () -> violations.add(ERROR_MESSAGE_EMPTY_ATTRIBUTE.formatted("timestamp"))
            );
```

Reply:
Nice suggestion!
Refactored this validation check.